blueprints/modular/about.yaml

```diff
@@ -0,0 +1,30 @@
+title: About
+'@extends': default
+
+form:
+  fields:
+    tabs:
+      fields:
+        features:
+          type: tab
+          title: About
+          fields:
+            header.details:
+              name: details
+              type: list
+              label: Details
+              fields:
+                .title:
+                  type: text
+                  label: Title
+                  validate:
+                    type: textarea
+                .icon:
+                  type: iconpicker

```

Comment:
This isn't showing previews for me, so it might be better to just use a text field.

Reply:
I'm trying to figure out a way to have them show up, I think its because grav is loading some older version of FA for itself. Will change it to iconpicker again after I figure it out